kzg_prover/src/circuits/tests.rs

```diff
@@ -85,26 +85,26 @@ mod test {
             full_prover(&params, &pk, circuit.clone(), vec![vec![]]);
 
         // Both the Custodian and the Verifier know what column range are the balance columns
+        // (The first column is the user IDs)
         let balance_column_range = 1..N_CURRENCIES + 1;
 
-        // The Custodian makes an opening at x = 0 for the Verifier
-        // (The first column is the user IDs)
-        let kzg_proofs = open_grand_sums::<N_CURRENCIES>(
+        // The Custodian makes multi-openings at x = 0 for the Verifier
+        let grand_sums_multi_proof = open_grand_sums::<N_CURRENCIES>(
             &advice_polys.advice_polys,
             &advice_polys.advice_blinds,
             &params,
             balance_column_range,
         );
 
-        // The Custodian creates a KZG proof of the 4th user balances inclusion
+        // The Custodian creates a KZG multi-proof of the 4th user ID & balances inclusion
         let user_index = 3_u16;
 
-        let balance_column_range = 1..N_CURRENCIES + 1;
-        let balance_opening_proofs = open_user_balances::<N_CURRENCIES>(
+        let column_range = 0..N_CURRENCIES + 1;
+        let openings_multi_proof = open_user_points::<N_CURRENCIES>(
```

Comment:
Again, what's inside this multi-proof?

Reply:
I will document all the new KZG functions in `utils` not to explain in details how the KZG proofs work in the test.